hca/api/templates/functions.jinja

```diff
@@ -0,0 +1,19 @@
+{% for class_info in classes %}
```

Comment:
Is the scary message `"""This file is autogenerated according to HCA api spec. DON'T TOUCH!"""` supposed to be here?